skyportal/tests/frontend/test_newsfeed.py

```diff
@@ -1,6 +1,6 @@
 import uuid
 import pytest
-
+from selenium.webdriver.common.action_chains import ActionChains
```

Comment:
Neat, going to start using this!

Reply:
Yeah starting to really look into it now... Using ActionChains fixed some flakiness in test_groups for me yesterday but doesn't seem like it's helping here...